wagtail/contrib/simple_translation/templates/simple_translation/admin/submit_translation.html

```diff
@@ -1,52 +1,27 @@
-{% extends "wagtailadmin/base.html" %}
-{% load i18n wagtailadmin_tags %}
-{% block titletag %}{{ view.get_title }} {{ view.get_subtitle }}{% endblock %}
-
-{% block content %}
-    {% include "wagtailadmin/shared/header.html" with title=view.get_title subtitle=view.get_subtitle icon="doc-empty-inverse" %}
-
-    <div class="nice-padding">
-        <form method="POST">
-            {% csrf_token %}
-
-            {% if next_url %}
-                <input type="hidden" name="next" value="{{ next_url }}">
+{% extends "wagtailadmin/base.html" %} {% load i18n wagtailadmin_tags %} {%
+block titletag %}{{ view.get_title }} {{ view.get_subtitle }}{% endblock %} {%
+block content %} {% include "wagtailadmin/shared/header.html" with
+title=view.get_title subtitle=view.get_subtitle icon="doc-empty-inverse" %}
+
+<div class="nice-padding" data-controller="w-bulk">
+    <form method="POST">
+        {% csrf_token %} {% if next_url %}
+        <input type="hidden" name="next" value="{{ next_url }}" />
+        {% endif %} {% for field in form.hidden_fields %}{{ field }}{% endfor %}
+
+        <ul class="fields">
+            {% block visible_fields %} {% for field in form.visible_fields %} {%
+            include "wagtailadmin/shared/field_as_li.html" %} {% endfor %} {%
+            endblock %} {% if form.show_submit %}
+            <li>
+                <input
+                    type="submit"
+                    value="{% trans 'Submit' %}"
+                    class="button"
+                />
+            </li>
             {% endif %}
```

Comment:
This template is somehow still incorrectly changed, only the last ~9 lines should be removed.